construct.yaml

```diff
@@ -51,25 +54,27 @@ specs:
   - mysql-client =8.0.27
   # Earlier versions of mysqlclient were build with older MySQL versions
   - mysqlclient >=2.0.3
-  - sqlalchemy
+  - sqlalchemy >=1.4.36
   - stomp.py =4.1.23
   # Middleware
-  - apache-libcloud
-  - boto3
-  - gfal2-util
+  - apache-libcloud >=3.5.1
+  - boto3 >=1.5.11
+  - gfal2 >=2.20.5
+  - gfal2-util >=1.7.1
   - fts3 >=3.12
-  - nordugrid-arc  # [not osx]
-  - python-gfal2
+  - nordugrid-arc >=6.15.1  # [not osx]
+  - python-gfal2 >=1.11.0
   # Constrain the version for now until using 9.1.3+ is understood
   # See emails about "HTCondor submission error" from the hackathon on 18/11/2021
-  - htcondor-utils 9.1.2 # [linux64]
-  - python-htcondor 9.1.2 # [linux64]
-  - rucio-clients
+  - htcondor-utils =9.0 # [linux64]
+  - python-htcondor =9.0 # [linux64]
```

Comment:
@fstagni Any objections to using htcondor `9.0.x` rather than `9.1.2`?

Conda-forge is only maintaining the latestest release and `9.0.x` releases and it's now preventing us from using a newer gct version (which then forces us to use older davix, gfal2 and more).

I could add `9.1.2` to the list of supported versions but I don't see any benifit to doing?

Reply:
Sure, especially considering https://github.com/DIRACGrid/DIRACOS2/pull/60#issuecomment-979165672